groovy-language-server/src/main/java/com/palantir/groovylanguageserver/CompilerWrapper.java

```diff
@@ -37,17 +39,31 @@
 
     /**
      * Returns a mapping from absolute path of source file to symbols located within these source files.
-     * @return the list of symbols
+     * @return the set of symbols
```

Comment:
can probably just delete the @return since it doesn't provide any additional info (and is actually wrong here...)